node-runner-cli/commands/dockercommand.py

```diff
@@ -90,7 +90,6 @@ def config(args):
     setupmode = SetupMode.instance()
     setupmode.mode = args.setupmode
     trustednode = args.trustednode if args.trustednode != "" else None
-    networkid = args.networkid if args.networkid != "" else None
```

Comment:
We would need this. For testnets, user can ignore to setup network id and code checks for `None`. So either we set here `None` or change the code where network id is set to None to check for empty string